src/Offer/Security/Permission/UserPermissionVoter.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace CultuurNet\UDB3\Offer\Security\Permission;
```

Comment:
Denk dat je deze beter ergens anders plaatst als je ze ook wil gebruiken om later de permissie voor bewerking van organizers te controleren bijvoorbeeld.

Niets in deze class of de use statements is trouwens rechtstreeks verbonden aan Offer.

Reply:
Ik heb de class gemoved naar CultuurNet\UDB3\Security\Permission, daar zitten alle security related classes.
Het lijkt me dat dan nog enkele classes zoals PermissionVoterInterface, CompositeVoter en GodUserVoter mogen verhuizen. Dat heb ik nog niet gedaan.